MAUI/DataForm/accessibility.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Accessibility in .NET MAUI DataForm control | Syncfusion
+description: Learn here about performing actions using accessibility in Syncfusion NET MAUI DataForm(SfDataForm) control in mobile and desktop applications from a single shared codebase.
```

Comment:
Learn here about Accessibility support with the Syncfusion .NET MAUI DataForm(SfDataForm) control in mobile and desktop applications from a single shared codebase.

Reply:
Changed.